schema.sql

```diff
@@ -0,0 +1,69 @@
+-- create database cataglog
+CREATE DATABASE IF NOT EXISTS catalog;
+
+-- Create the Genre table
+CREATE TABLE genre (
+    id SERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL,
+);
+
+-- Create the Item table
+CREATE TABLE item (
+    id SERIAL PRIMARY KEY,
+    genre VARCHAR(255) NOT NULL,
+    author VARCHAR(255) NOT NULL,
+    source VARCHAR(255) NOT NULL,
+    label VARCHAR(255) NOT NULL,
+    publish_date DATE(255) NOT NULL,
+    archived BOOLEAN(255) NOT NULL,
+    can_be_archived BOOLEAN(255) NOT NULL,
+    genre_id INT REFERENCES genre(id)
+);
```

Comment:
When creating SQL tables, there isn't a mechanism to portray inheritance between the tables, and so the better way to do this is to remove the item table, and add its properties to the tables of the classes that inherit from it. author, genre and label should be considered as foreign keys in the music_album, book, and game tables.
Also, can_be_archived property is not present in the class diagram as a property, so it should be removed from all tables.